probe/probe.go

```diff
@@ -302,3 +329,347 @@ func probeNode(c *k8s.Client) {
+	color.Green(kd.KernelVersion)
+	fmt.Printf("\t Kubelet Version: ")
+	color.Green(kd.KubeletVersion)
+	fmt.Printf("\t Container Runtime: ")
+	color.Green(kd.ContainerRuntime)
+	fmt.Printf("\t Active LSM: ")
+	color.Green(kd.ActiveLSM)
+	fmt.Printf("\t Host Security: ")
+	color.Green(strconv.FormatBool(kd.HostSecurity))
+	fmt.Printf("\t Container Security: ")
+	color.Green(strconv.FormatBool(kd.ContainerSecurity))
+	fmt.Printf("\t Container Default Posture: \n")
+	fmt.Printf("\t\t File Action: ")
+	color.Green(kd.ContainerDefaultPosture.FileAction)
+	fmt.Printf("\t\t Capabilities Action: ")
+	color.Green(kd.ContainerDefaultPosture.CapabilitiesAction)
+	fmt.Printf("\t\t Network Action: ")
+	color.Green(kd.ContainerDefaultPosture.NetworkAction)
+	fmt.Printf("\t Host Default Posture: \n")
+	fmt.Printf("\t\t File Action: ")
+	color.Green(kd.HostDefaultPosture.FileAction)
+	fmt.Printf("\t\t Capabilities Action: ")
+	color.Green(kd.HostDefaultPosture.CapabilitiesAction)
+	fmt.Printf("\t\t Network Action: ")
+	color.Green(kd.HostDefaultPosture.NetworkAction)
+}
+
+// sudo systemctl status kubearmor
+func isSystemdMode() bool {
+	cmd := exec.Command("systemctl", "status", "kubearmor")
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		if _, ok := err.(*exec.ExitError); ok {
+			return false
+		} else {
+			return false
+		}
+	}
+	color.Green("\nFound KubeArmor running in Systemd mode \n\n")
+	return true
+}
+
+func probeSystemdMode() error {
+	var kd *KubeArmorProbeData
+	jsonFile, err := os.Open("/tmp/karmorProbeData.cfg")
+	if err != nil {
+		log.Println(err)
+		return err
+	}
+
+	buf, err := io.ReadAll(jsonFile)
+	if err != nil {
+		log.Println("an error occured when reading file", err)
+		return err
+	}
+	var json = jsoniter.ConfigCompatibleWithStandardLibrary
+	err = json.Unmarshal(buf, &kd)
+	if err != nil {
+		log.Println("an error occured when parsing file", err)
+		return err
+	}
+	_, err = boldWhite.Printf("Host : \n")
+	if err != nil {
+		color.Red(" Error")
+	}
+	printKubeArmorProbeOutput(kd)
+	return nil
+}
+
+func getAnnotatedPods(c *k8s.Client) error {
+	// Annotated Pods Description
+
+	pods, err := c.K8sClientset.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	_, err = boldWhite.Printf("Annotated pods : \n")
+	if err != nil {
+		color.Red(" Error printing bold text")
+	}
+	fmt.Printf("\t NAMESPACE \t\t NAME \t\t POLICY\n")
+	policyMap, err := getPoliciesOnAnnotatedPods(c)
+	if err != nil {
+		color.Red(" Error getting policies on annotated pods")
+	}
+
+	for _, p := range pods.Items {
+		if p.Annotations["kubearmor-policy"] == "enabled" {
+			v, err := c.K8sClientset.CoreV1().Pods(p.Namespace).Get(context.Background(), p.Name, metav1.GetOptions{})
+			if err != nil {
+				return err
+			}
+			for key, value := range v.Labels {
+				for x, y := range policyMap {
+					if slices.Contains(y, key+":"+value) {
+						fmt.Println("\t", v.Namespace, "\t\t ", v.Name, "\t\t ", x)
+					}
```

Comment:
How about we push it to an array and leverage https://github.com/olekukonko/tablewriter for printing the table at the end?